.github/workflows/ci.yml

```diff
@@ -8,98 +8,53 @@ on:
 jobs:
   build:
     name: Build and test
-
-    runs-on: ${{ matrix.os }}
-
+    runs-on: ubuntu-latest
     env:
       EMAIL_RATE_LIMIT: 3600
       MATCHER_MAX_CAPACITY: 80
       MYSQL_DATABASE: test_econ
       MYSQL_ROOT_PASSWORD: password
-      SIHL_ENV: test
-      TEST_EMAIL: test@econ.uzh.ch
-
     services:
       database-root:
         image: mariadb:10.6
         env:
           MYSQL_DATABASE: ${{ env.MYSQL_DATABASE }}
           MYSQL_ROOT_PASSWORD: ${{ env.MYSQL_ROOT_PASSWORD }}
-        ports: ["3306:3306"]
         options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
 
       database-tenant:
         image: mariadb:10.6
         env:
           MYSQL_DATABASE: ${{ env.MYSQL_DATABASE }}
           MYSQL_ROOT_PASSWORD: ${{ env.MYSQL_ROOT_PASSWORD }}
-        ports: ["3307:3306"]
+          MYSQL_TCP_PORT: 3307
```

Comment:
Are these variables unused? -> remove

Reply:
This was taken from the previous setup too. As I don't see a point in why the port has to be changed, I updated the code, so both services run on the default port.